src/invidious/comments.cr

```diff
@@ -144,7 +144,8 @@ def fetch_youtube_comments(id, cursor, format, locale, thin_mode, region, sort_b
 
               content_html = node_comment["contentText"]?.try { |t| parse_content(t) } || ""
               author = node_comment["authorText"]?.try &.["simpleText"]? || ""
-
+              verified = (node_comment["authorCommentBadge"]? != nil)
+              json.field "verified", (verified || false)
```

Comment:
you can inline the `!= nil` statement with the `json.field` line. Also note that you don't need to fallback to `false` (using the (`||`) operator), as your verified wheck will always be boolean.

And please keep the existing blank lines between code chunks, to improve readibility :)